Ronni-ronsbons/week-3/hw giffaw-master/scripts/app.js

```diff
@@ -0,0 +1,80 @@
+$(document).ready(function() {
+  // Upon page load, show Top 25 gifs
+  let key = 'zbimCMBPhJzgKJkfTxlFf4tGndzeAzO2';
+
+  // Trending API
+  $.ajax({
+    method: 'GET',
+    url: `http://api.giphy.com/v1/gifs/trending&api_key=${key}`,
+    success: gifSuccess,
+    error: gifError,
+  });
+  // RETURNS ERROR 401 - 'ACCESS TO XMLHTTPREQUEST from origin "null" has been blocked by CORS policy: No "Access-Control-Allow-Origin" header is present on the requested resource.
```

Comment:
You were receiving the cors error due to your http call being http not https. This is because the server the is sending the data is a secure protocol.